src/frontend/screens/Settings/sections/LogSettings/components/UploadedLogFilesList/index.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { memo, useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Launch, Delete } from '@mui/icons-material'
+
+import {
+  Dialog,
+  DialogContent,
+  DialogHeader
+} from 'frontend/components/UI/Dialog'
+import SvgButton from 'frontend/components/UI/SvgButton'
+
+import useUploadedLogFiles from 'frontend/state/UploadedLogFiles'
+import type { UploadedLogData } from 'common/types'
+import {
+  useGlobalState,
+  useShallowGlobalState
+} from 'frontend/state/GlobalStateV2'
+
+import './index.css'
+
+interface UploadedLogFileItemProps extends UploadedLogData {
+  url: string
+}
+const UploadedLogFileItem = memo(function UploadedLogFileItem(
+  props: UploadedLogFileItemProps
+) {
+  const { url, name, uploadedAt } = props
+  const { t } = useTranslation()
+
+  const uploadedAgoText = useMemo(() => {
+    const minutesAgo = Math.round((Date.now() - uploadedAt) / 1000 / 60)
+    const showHours = minutesAgo > 60
+    if (showHours) {
+      return t(
+        'setting.log.upload.hours-ago',
+        'Uploaded {{hoursAgo, relativetime(hours)}}',
+        { hoursAgo: -Math.round(minutesAgo / 60) }
+      )
+    }
+    return t(
+      'setting.log.upload.minutes-ago',
+      'Uploaded {{minutesAgo, relativetime(minutes}}',
```

Comment:
```suggestion
      'Uploaded {{minutesAgo, relativetime(minutes)}}',
```

should this be like this? comparing with the previous string

Reply:
That was indeed incorrect. Interestingly enough, it still worked just fine.